entries/k/koho.ca.json

```
@@ -0,0 +1,14 @@
+{
+  "Koho": {
+    "domain": "koho.ca",
+    "tfa": [
+      "sms"
+    ],
```

As @hkamran80 already mentioned, it sounds like what KOHO is offering is two-step __authorization__ and not two factor __authentication__. Because of that, please change replace the `tfa` element with the following `contact` element.
```suggestion
    "contact": {
      "twitter": "GetKoho",
      "facebook": "KOHO",
      "form": "https://www.koho.ca/contact/",
    },
```